tracpro/polls/tasks.py

```diff
@@ -130,3 +131,39 @@ def org_task(self, org):
         are no longer on the remote.
         """
         apps.get_model('polls', 'Poll').objects.sync(org)
+
+
+@task
+def sync_questions_categories(org, polls):
+    # Create new or update SELECTED Polls to match RapidPro data.
+    from tracpro.polls.models import Poll, Question
+
+    # Save the associated Questions for this poll here
+    # now that these polls have been activated for the Org
+    selected_poll_names, selected_polls = [], []
+    for poll in polls:
+        selected_poll_names.append(poll.name)
+        selected_polls.append(Poll.objects.get(id=poll.id))
```

Comment:
Ok, I think I know how to replace this with something a little simpler. But we still need to check that the  poll we get back from temba is in the poll list we want to sync. Let me know if my change makes sense! (coming soon)